### PR TITLE
content: cleanup TOB information

### PR DESCRIPTION
**Pull request by cyphar**
Explain in a bit more detail how the TOB is chosen, as well as what the
TOB Chair is. In addition, put the terms in a separate "subtitle"
section of each card.

Signed-off-by: Aleksa Sarai <cyphar@cyphar.com>

### CODE REVIEW

**Comment by cyphar** [comment]
I'm not sure if this layout is acceptable -- @celestehorgan how do you think term dates should be shown?

2020-05-30-014540_1118x593_scrot

File: layouts/shortcodes/card.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 {{ $name := .Get "name"}}
 {{ $company := .Get "company" }}
 {{ $title := .Get "title" }}
+{{ $subtitle := .Get "subtitle" }}
```
Comment (celestehorgan) [inline layouts/shortcodes/card.html:5]: I'd name this something more meaningful, i.e `$termdate`.

File: layouts/shortcodes/card.html [hunk 2]
```diff
@@ -20,6 +21,9 @@ <h5>{{ $company }}</h5>
   {{ with $title }}
     <h5>{{ $title }}</h5>
   {{ end }}
+  {{ with $subtitle }}
+    <h6>{{ $subtitle }}</h6>
```
Comment (celestehorgan) [inline layouts/shortcodes/card.html:25]:
```suggestion
    <h6>Term: {{ $subtitle }}</h6>
```

**Comment by cyphar** [comment]
Here's the new version. I decided to make it smaller than `h5` (and enlarge `h5`) so that we don't have three lines of text with different meanings that are all the same size.

2020-05-30-043147_1127x632_scrot

**Comment by celestehorgan** [comment]
@cyphar You've got a merge conflict! Mind resolving it? :)

**Comment by cyphar** [comment]
Rebased, @celestehorgan.

**Pull request merged**